Purchases/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -54,8 +55,13 @@ class PurchasesOrchestrator {
     private let identityManager: IdentityManager
     private let receiptParser: ReceiptParser
     private let deviceCache: DeviceCache
+    private let manageSubscriptionsModalHelper: ManageSubscriptionsModalHelper
+    private let beginRefundRequestHelper: BeginRefundRequestHelper
     private let lock = NSRecursiveLock()
 
+    @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+    lazy var storeKit2Listener = StoreKit2TransactionListener(delegate: self)
```

Comment:
I think this could be `internal private(set)` to ensure it's not changed from outside the type?

Reply:
Bump?

Reply:
sorry I forgot to reply on this one, but it's the same story as the other one - not sure how best to do this in a way that's still compatible with tests

Reply:
I figured when I saw your other comment hehe. Sounds good.